src/pages/UserPage/UserPage.jsx

```diff
@@ -135,41 +258,52 @@ const UserPage = () => {
               <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
               {errors.password && <p className='restaurantAdminView__error'>{errors.password}</p>}
             </span>
-            <label className='userPage__form-label' htmlFor='userPhoneNumber'>Phone Number</label>
-            <span>
-              <input
-                className='userPage__form-input'
-                type='number'
-                id='userPhoneNumber'
-                name='phoneNumber'
-                disabled={!isEditable}
-                onChange={handleInputChange}
-                placeholder={user.phoneNumber}
-                value={formData.phoneNumber}
-              />
-              <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
-              {errors.phoneNumber && <p className='restaurantAdminView__error'>{errors.phoneNumber}</p>}
-            </span>
-            <label className='userPage__form-label' htmlFor='userAddress'>Address</label>
-            <span>
-              <input
-                className='userPage__form-input'
-                type='text'
-                name='address'
-                id='userAddress'
-                disabled={!isEditable}
-                onChange={handleInputChange}
-                placeholder={user.address}
-                value={formData.address}
-              />
-              <RiEdit2Fill onClick={handleEditClick}  className='userPage__form-icon'/>
-              {errors.address && <p className='restaurantAdminView__error'>{errors.address}</p>}
-            </span>
-            <label className='userPage__form-label' htmlFor='userCity'>City</label>
+            {user.phone_numbers && user.phone_numbers.map((phoneNumber, index) => (
+              <div key={phoneNumber.id_user_phone_number}>
+                <label className='userPage__form-label' htmlFor='userPhoneNumber'>{languageSelector(language, 'signInPhone')} {index + 1}</label>
+                <span>
+                <input
+                  className='userPage__form-input'
+                  type='text'
+                  id='userPhoneNumber'
+                  name={`phone_numbers[${index}][phone_number]`}
+                  disabled={!isEditable}
+                  onChange={handleInputChange}
+                  placeholder={phoneNumber.phone_number}
+                  value={formData.phone_numbers[index].phone_number}
```

Comment:
is this the formData necessary? why not using the resources of mapped object?

Reply:
Given that this is a critical change, let's move it to resolved, but I consider this would require an enhancement.